packages/components/src/Form/Inputs/InputColor/utils/stringToSimpleHsv.ts

```diff
@@ -29,5 +29,17 @@ import { SimpleHSV } from '../types'
 
 export const stringToSimpleHsv = (color: string): SimpleHSV => {
   const hsvColor = hsv(color)
-  return { h: hsvColor.h, s: hsvColor.s, v: hsvColor.v }
+
+  /**
+   * The hsv helper function returns NaN for hueless and saturationless values
+   * like black, white, grey. We will instead send 0 if either value is NaN for use
+   * when calculating handle positioning and coloring on the hue slider and saturation preview.
+   *
+   * 0 is used as a fallback value to mirror MDN's handling of hueless and saturationless values:
+   * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color
+   */
+  const hue = isNaN(hsvColor.h) ? 0 : hsvColor.h
+  const saturation = isNaN(hsvColor.s) ? 0 : hsvColor.s
+
+  return { h: hue, s: saturation, v: hsvColor.v }
```

Comment:
The terse-obsessed nerd in me see this (completely unneeded) refactor to tighten things up a bit here:

```suggestion
  let { h, s, v } = hsv(color)
  /**
   * The hsv helper function returns NaN for hueless and saturation-less values
   * like black, white, grey. We will instead send 0 if either value is NaN for use
   * when calculating handle positioning and coloring on the hue slider and saturation preview.
   *
   * 0 is used as a fallback value to mirror MDN's handling of hueless and saturation-less values:
   * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color
   */
  if(isNaN(h)) h = 0
  if(isNaN(s)) s = 0

  return { h, s, v }
```